### DISCUSSION TITLE
Any way to use arbitrary metadata (keywords, etc.)?

### DISCUSSION THREAD

**Discussion by wendyxiou** [knowledgefutures/pubpub#1308] [Support]
Hi!  I love pubpub  :)   Just have a quick question.  The pubpub page says that it "Include(s) article & collection-level metadata for easier organization of content and improved discovery."  I assumed that this means arbitrary metadata for things like keywords to help discovery for search engines and stuff, but I can't figure out how to add my own metadata.

Does anyone know how I can add article-level metadata, either the "right" way, or any suggestions on how I can do the equivalent of adding keyword metadata sneakily, like how people embed keywords in tiny font in white text at the bottom of craigslist ads and stuff like that. Even a total jerryrig solution like that would be so rad.  Thank you :)

**Comment by gabestein** [comment]
Thanks so much for the kind words and the question.

We are definitely planning to support adding arbitrary metadata to collections and their Pubs! But that's at least a few months away, and right now we only support a limited set that's effectively for citation and depositing of specific types of content (books, journals, etc.). For the most part, we see folks just structure metadata using text in the editor, e.g. [Technology, Mind and Behavior](https://tmb.apaopen.org/pub/nonverbal-overload/release/1). Most search engines, including academic aggregators, are full-text these days, so that typically gets those fields picked up.

As for doing it unobtrusively, you could totally use the supertext or subtext options at the bottom of the page, or probably even put stuff in a footnote, which I believe most search engines will index (although because it's collapsed by default, that may depend some on the specific engine).

I hope that's helpful, and I'd also be interested to hear what other creative things folks are doing to add this kind of data.

**Comment by aimeediehl** [comment]
Hi! I have the same question and noticed this topic was still open. We just published our first article and I could use some help understanding precisely what article-level metadata gets extracted automatically before I mint the DOI. For example, we have author-suggested keywords but our librarians have suggested a few more. Any updates on this, or explainers for newbies? TIA.
